package.json

```diff
@@ -40,13 +40,17 @@
   "dependencies": {
     "@apidevtools/json-schema-ref-parser": "^11.1.0",
     "@openapi-contrib/openapi-schema-to-json-schema": "^5.1.0",
+    "@types/lodash.camelcase": "^4.3.9",
```

Comment:
types should stay in the `devDependencies`

Reply:
right, missed it